#Raptor HeartBeat failure

Header header
bool RaptorHBFail
uint8 counter